src/status_im2/contexts/communities/discover/events.cljs

```diff
@@ -0,0 +1,45 @@
+(ns status-im2.contexts.communities.discover.events
+  (:require [clojure.set :as set]
+            [clojure.string :as string]
+            [taoensso.timbre :as log]
+            [utils.re-frame :as rf]))
+
+(defn camel->kebab
+  [k]
+  (let [s                  (name k)
+        lower-cased        (string/lower-case
+                            (string/replace s #"([a-z])([A-Z])" "$1-$2"))
+        starts-with-digit? (re-matches #"^\d.*" s)
+        predicate?         (some #(string/starts-with? lower-cased %)
+                                 ["can-" "is-"])]
+    (cond starts-with-digit? s
+          predicate?         (keyword (str lower-cased "?"))
+          :else              (keyword lower-cased))))
+
+(defn rename-keys
+  [m]
+  (reduce (fn [acc [k v]]
+            (let [new-key (if (keyword? k) (camel->kebab k) k)]
+              (cond
+                (map? v) (assoc acc new-key (rename-keys v))
+                :else    (assoc acc new-key v))))
+          {}
+          m))
+
+(rf/defn handle-contract-communities
+  {:events [:fetched-contract-communities]}
```

Comment:
`:communities/fetched-contract-communities`

Reply:
yeah, but why it isn't `:communities/contract` and why is it contract btw ?

Reply:
The distinction between communities and contract-communities was explained to me [here](https://github.com/status-im/status-mobile/issues/16015#issuecomment-1587763141):

> They should all be coming from the contract (@cammellos)

and

> The communities portal operates independently of the app/account state. The contents of the featured section are not influenced by whether a community has been joined or not.
> 
> For context: There exists a distinct DApp, called 'Community Directory Curation' which is responsible for managing the communities to be listed on the App's Community Portal. This DApp is self-contained and does not interact with Status clients. Essentially, users utilize their voting power, via SNT tokens, to vote for or against the inclusion of a particular community in the directory and the featured section. All pertinent data is housed within the contract, and this contract serves as the sole input to the app. To simplify, data sourced from the contract is structured as follows: (@osmaczko)